src/main/java/uk/gov/justice/hmpps/prison/web/config/SwaggerConfig.java

```diff
@@ -10,9 +11,9 @@
 import springfox.bean.validators.configuration.BeanValidatorPluginsConfiguration;
 import springfox.documentation.builders.PathSelectors;
 import springfox.documentation.builders.RequestHandlerSelectors;
-import springfox.documentation.service.ApiInfo;
-import springfox.documentation.service.Contact;
+import springfox.documentation.service.*;
```

Comment:
We favor explicit imports over wildcards, it takes some convincing to make your IDE accept that though :D

Reply:
done